cdap-docs/admin-manual/source/operations/logging.rst

```diff
@@ -410,6 +371,79 @@ programs of an application and system services with the logging framework:
     **CDAP Logging Framework:** From YARN containers, through Kafka and the Log Saver
     Service, to HDFS
 
+- Logs are collected from individual programs running in YARN containers.
+
+- YARN writes the log messages emitted by containers to files inside the container.
+
+- In addition, CDAP programs publish these messages to Kafka.
+
+- CDAP System Services run (depending on the service) either on cluster edge nodes or in
+  YARN containers. Where they run determines the file that configures that service's
+  logging.
+
+- The Log Saver Service (*Log.saver*) is configured to read log messages for the
+  ``logs.user-v2`` Kafka topic (set by the property ``log.kafka.topic``). The number of
+  log saver instances can be scaled to process the Kafka partitions in parallel, if
+  needed.
+  
+  *Log.saver*, by default, runs only the CDAP Log Pipeline: it reads the messages from
+  Kafka, groups them by program or application, buffers and sorts them in memory, and
+  finally persists them to files in HDFS. Each of these files corresponds to one program
+  or application, depending on how the grouping is configured. (This is set by the
+  property :ref:`log.publish.partition.key <logging-monitoring-writing-logs-to-kafka>`,
+  described :ref:`below <logging-monitoring-configuring-the-log-saver-service>`.)
+
+  *Note:* These files are configured to rotate based on time and size; they can be
+  changed using the properties ``log.pipeline.cdap.file.max.size.bytes`` and
+  ``log.pipeline.cdap.file.max.lifetime.ms`` in the :ref:`cdap-site.xml
+  <appendix-cdap-default-logging>` file as described in 
+  :ref:`logging-monitoring-log-pipeline-configuration`.
+
+- For security, the files written out to persistent storage in HDFS have permissions set
+  such that they are accessible only by the ``cdap`` user.
+
+- In addition, custom log pipelines can be configured by adding an XML file in a
+  prescribed location. Each pipeline buffers log messages in memory and sorts them based
+  on their timestamp. 
+ 
+- In addition to persisting logs to files, the *Log.saver* also emits metrics about the
+  number of log messages emitted by each program. These metrics can be retrieved by
+  querying the :ref:`CDAP metrics system <http-restful-api-metrics>`.
+  
+  These tables list the metrics from the section :ref:`available-system-metrics` of the
+  :ref:`http-restful-api-metrics`. See that section for further information.
+  
+  .. These tables are copied from cdap-docs/reference-manual/source/http-restful-api/metrics.rst
+  .. and are not guarded.
+
+  .. list-table::
+     :widths: 50 50
+
+     * - **Service Metric**
+       - **Description**
+     * - ``system.requests.count``
```

Comment:
The last two are probably the only ones relevant to users, even if the others are valid.